spec/requests/api/v3/work_packages/create_resource_spec.rb

```diff
@@ -310,7 +310,7 @@
           expect(last_response.body).to be_json_eql(
             api_v3_paths.file_links(work_package.id).to_json
           ).at_path("_links/fileLinks/href")
-          expect(last_response.body).to be_json_eql("1").at_path("_embedded/fileLinks/total")
+          expect(last_response.body).not_to have_json_path("_embedded/fileLinks")
```

Comment:
🟡 I'd rather remove this line, instead of checking for absence. As we no longer CAN embed it, why should we check for it to not be embedded?

Also, looking at the test here, should we maybe add an expectation, that the link `addFileLinks` is available? Equivalent to what we do with the link for the collection in 310.

Reply:
It's tricky to test against something, not to be added in the future. But makes sense.